code/ATMOSPHERICS/pipes.dm

```diff
@@ -27,7 +27,10 @@
 		air_update_turf()
 
 /obj/machinery/atmospherics/pipe/return_air()
-	return parent.air
+	if(parent)
+		return parent.air
+	else
+		return null

```

Comment:
On top of that, most of Atmos' runtimes are deliberately tied to scenarios we don't want, meaning the runtime is our sign that something <i>else</i> is broken.